### DISCUSSION TITLE
希望增加些功能

### DISCUSSION THREAD

**Discussion by r233r** [hifun-team/hexo-theme-miracle#6] [Ideas]
希望能增加些功能：

- 音乐播放器
- 页脚备案
- LaTeX 公式
- Mermaid 图

**Comment by ocoke** [comment]
十分感谢您的支持，页脚备案与 Mermaid 图已支持。

**Comment by ocoke** [comment]
我们不太推荐在你的网站上加入音乐，特别是自动播放。

您可以使用 Hexo 5 注入器 加入此功能，详见：[通过 Hexo 5 注入器为主题添加新功能](https://hexo.fluid-dev.com/posts/hexo-injector/)
